src/locales/en.json

```diff
@@ -231,6 +231,43 @@
     "aboutUsHeader": "About us",
     "aboutUsText": "We are a group of multi-national youth who found their common ground in using tech for the benefit of people. Juniper, consists of a group of 5 junior developers and 1 lead engineer working together to educate children in MENA about environmental issues and climate change through educational games. Our target group is kids living in MENA that are 5-9 years old; thus, Juniper is delivered in three main languages to help kids and their teachers/parents to easily surf the website.",
     "whyWeDoItHeader": "Why we do it",
-    "whyWeDoItText": "We have a moral obligation to preserve nature and its features, because protecting the environment protects humanity from toxic substances and our quality of food. We believe that the new generation in MENA should be supported with the right tools to believe and act for the benefit of humanitarian and social issues, on the local and global scale. Given the right tools, it is not impossible to raise a generation of activists."
+    "whyWeDoItText": "We have a moral obligation to preserve nature and its features, because protecting the environment protects humanity from toxic substances and our quality of food. We believe that the new generation in MENA should be supported with the right tools to believe and act for the benefit of humanitarian and social issues, on the local and global scale. Given the right tools, it is not impossible to raise a generation of activists.",
+    "teamInfo": {
+      "louis": {
+        "name": "Louis Li",
+        "bio": "Louis is the coding instructor in Re:Coded Turkey. With a lot of exciting stories and impressive experience he creates the butterfly effect.",
+        "role": "Project Engineer"
+      },
+      "ammar": {
+        "name": "Ammar Sammour",
+        "bio": "Ammar is the Coding Instructor at Front-end Web Development Bootcamp in Turkey provided by Re:Coded! He spends his time coaching our fellows to become web development ninjas!",
+        "role": "Lead Engineer"
+      },
+      "ibrahim": {
+        "name": "Ibrahim Aljarad",
+        "bio": "Hey, I'm Ibrahim from Syria, I studied coding during the frontend web developement bootcamp provided by Re:Coded where Flatiron School's curriculum was taught, I also studied economics at Alfurat university. My other experiences include working with several channels and agencies as a satellite producer providing the technical support for the field producing teams.",
```

Comment:
typo in development
capitalize Alfurat University

Reply:
PTAL